02/app.js

```diff
@@ -1 +1,9 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+const dataUrlList = document.querySelectorAll('[data-url]');
+const urlsList = [];
```

Comment:
Czy faktycznie należało użyć dodatkowej zmiennej (i pętli)? 
Nie łatwiej będzie to zrobić "od razu"?